tests/functional/api/test_bulk.py

```diff
@@ -35,9 +43,12 @@ def test_it_raises_errors_for_invalid_request(self, app, lms_auth_header):
     def test_it_accepts_a_valid_request(
         self, app, nd_json, lms_auth_header, db_session
     ):
-        app.post("/api/bulk", nd_json, headers=lms_auth_header)
```

Comment:
I _assume_ this was an error. I can't think of a good reason to make the request twice?